scanpipe/templates/scanpipe/resource_detail.html

```diff
@@ -34,15 +34,19 @@
       </ul>
     </div>
 
-    <section class="mx-5">
-      <div class="message-header">
+    <section class="mx-5 message is-dark">
+      <div class="message-header is-flex">
         <p>
           {{ object.name }}
           | {{ object.size|filesizeformat }}
           | <a class="has-text-white" href="{{ object.get_raw_url }}?as_attachment=1"><i class="fas fa-download"></i></a>
           <br>
           <span style="font-size: 10px;">{{ object.path }}</span>
         </p>
+        <div>
+          <button class="button prev-btn is-dark" disabled onclick="previousValue();"><i class="fas fa-arrow-up"></i>&nbsp;Prev</button>
```

Comment:
Inline JavaScript is not consistent with the rest of the codebase, keep the function but assign the click action using the `addEventListener`.